baremetal_network_provisioning/bnpclient/bnp_client_ext/bnpswitch/_bnp_switch.py

```diff
@@ -111,41 +109,34 @@ class BnpSwitchUpdate(extension.ClientExtensionUpdate, BnpSwitch):
     allow_names = False
 
     def add_known_arguments(self, parser):
-        parser.add_argument(
-            '--access_protocol',
-            help=_('Protocol with which the Switch will be connected'))
-        parser.add_argument(
-            '--access_parameters',
-            metavar=meta, action='append', dest='access_parameters',
-            type=utils.str2dict,
-            help=_('SNMP Credentials of the Switch'))
-        utils.add_boolean_argument(
-            parser, '--enable',
-            help=_('Enable or Disable the switch'))
-        parser.add_argument(
-            '--rediscover', action='store_true',
-            help=_('Trigger rediscovery of the Switch'))
+
+        #parser.add_argument('name', metavar='NAME',help=_('Name of the physical switch'))
```

Comment:
why this line is commented? as per the DB schema we have supported for including the name of the switch during creation .